docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -118,13 +125,20 @@ Each target can have one or more [compilations](#compilations).
-| `artifactsTaskName` | The name of the task that builds the resulting artifacts of this target.                                                                          |
-| `components`        | The components used to setup Gradle publications.                                                                                                 |
+| **Name**            | **Description**                                                                                                                                                                                                                                                                              | 
+|---------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `attributes`        | Attributes used for [disambiguating targets](multiplatform-set-up-targets.md#distinguish-several-targets-for-one-platform) for a single platform.                                                                                                                                            |
+| `preset`            | The preset that the target has been created from, if any.                                                                                                                                                                                                                                    |
+| `platformType`      | Designates the Kotlin platform of this target. Available values: `jvm`, `androidJvm`, `js`, `native`, `common`.                                                                                                                                                                              |
+| `artifactsTaskName` | The name of the task that builds the resulting artifacts of this target.                                                                                                                                                                                                                     |
+| `components`        | The components used to setup Gradle publications.                                                                                                                                                                                                                                            |
+| `compilerOptions`   | The [compiler options](gradle-compiler-options.md) used for the target. This declaration overrides any `compilerOptions` configured at [top level](multiplatform-dsl-reference.md#top-level-blocks). To use it, add the following opt-in: `@OptIn(ExperimentalKotlinGradlePluginApi::class)` |
```

Comment:
> This declaration overrides any `compilerOptions` configured at [top level](multiplatform-dsl-reference.md#top-level-blocks).

Would be nice to clarify that the task level explicitly configured compilerOptions override top level and target level ones. Something like we have here, maybe https://github.com/JetBrains/kotlin-web-site/blob/fa8d26d84a18a8f04fe78dfb9550801e058b43e5/docs/topics/gradle/gradle-compiler-options.md#L85

Reply:
I discussed with Yahor to only cover top-level and target-level compilerOptions for now and add documentation about task-level a bit later.